Video/tormyvancool_YOUTUBE Downloader.lua

```diff
@@ -0,0 +1,367 @@
+-- @description YOUTUBE Downloader
+-- @author Tormy Van Cool
+-- @version 2.9
+-- @changelog
+--   1.0 2024-26-10
+--       # First Release
+--   1.1 2024-26-10
+--       + Processes Notifications
+--       - /Video/
+--       + /Videos/
+--   1.2 2024-26-10
+--       - --merge-output-format mp4
+--       + -S vcodec:h264,res,acodec:aac
+--   1.3 2024-26-10
+--       - 10
+--       + 2
+--   1.4 2024-26-10
+--       - 2
+--       + 5
+--   1.5 2024-26-10
+--       - 5
+--       + 1
+--       # Unified Update
+--   1.6 2024-26-10 - 1
+--       + 2
+--       + Version
+--   1.7 2024-27-10
+--       - 'start "" "' from all O.S.s
+--       + 'start "UPDATE & DOWNLOAD" "' Win
+--   1.8 2024-27-10
+--       - GGGGG = ''
+--       - 1
+--       + Start = '"'
+--       + 2
+--   1.9 2024-27-10
+--       + Check saved project
+--       - 1
+--       + 2
+--   2.0 2024-27-10
+--       - "chmod +x " ..  MainPath
+--       + 'chmod +x "' ..  MainPath .. '"'
+--       # Ordered Variables
+--       - 2
+--       + 1
+--       + Apple Trial
+--   2.3 2024-27-10
+--       # Linux execution correction
+--       + Credits
+--       # 2.1 and 2.2 just trials due issues with Linux and Apple
+--   2.31 2024-28-10
+--       # Binaries directly form the source
+--   2.32 2024-28-10
+--       - yt-dlp
+--       + yt-dlp_linux
+--   2.4 2024-29-10
+--       # Adjusted header style for production
+--   2.5 2024-11-04
+--       - Various 
+--       + VideoPath = 'Video'
+--   2.6 2024-11-05
+--       + check for temrination of temporary file upfrotn import the video
+--   2.7 2024-11-05
+--       - Check Routine
+--   2.8 2024-11-06
+--       + Detects Nework Interruptions during download
+--       + Removes leftovers
+--       + URLs as filename: forbidden
+--       + Limitation to only alphanumerical characters
+--   2.9 2024-11-06
+--       + Check IfFileExists: Overwrite, Newname, Exit
+-- @provides
+--   [windows] yt-dlp/yt-dlp.exe https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp.exe
+--   [linux] yt-dlp/yt-dlp_linux https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp
+--   [darwin] yt-dlp/yt-dlp_macos https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_macos
```

Comment:
Should un-list for unsupported 32-bit Windows/macOS, and for the script itself too so that it cannot be installed where it can't be used.

Reply:
For the unsupported 32bits I wil modify what's written between `[]` but how to avoid ppl to download the script itself on 32bit machines: not any clue.

Reply:
Repeat the `provides` line for the script itself for each platform (dot = shortcut for the current file):

```lua
-- @provides
--  [win64]        .
--  [darwin64]     .
--  [darwin-arm64] .
--  [linux]        .
```

Reply:
Ok I will.

Reply:
Modified as indicated

Reply:
Committed